tdvt/tdvt/test_results.py

```diff
@@ -159,7 +169,16 @@ def parse_default_test_cases(self):
         #Parse the setup file to get the count.
         if not self.test_case_map and self.test_set:
             if self.test_set.is_logical:
-                test_result = TestCaseResult("Not run", 0, "", 0, "Not run", self.error_status, None, self.test_config)
+                if self.test_set.test_is_enabled is False:
```

Comment:
It would be nice if you could combine this if/else section with the one below (191 to 204) into a new function. Not required thought if it's difficult.